modules/dag-shared/src/main/scala/org/tessellation/dag/package.scala

```diff
@@ -19,6 +19,7 @@ package object dag {
     classOf[SnapshotOrdinal] -> 602,
     classOf[DAGBlock] -> 603,
     classOf[BlockReference] -> 604,
-    classOf[GlobalSnapshotInfo] -> 605
+    classOf[GlobalSnapshotInfo] -> 605,
+    classOf[ActiveTip] -> 606
```

Comment:
Don't you need the `DeprecatedTip` and `GlobalSnasphotTips' as well?

Reply:
We'll see later I guess :)

Reply:
It's needed of course. Fixing.